README.md

```diff
@@ -1 +1,28 @@
-# matter-handbook
\ No newline at end of file
+# matter-handbook
+
+## Author guidelines
+This is an informal set of guidelines for authors, where we can document the conventions that are being used in this handbook.
+
+### Connectivity Standards Alliance Naming
+When referring to the Alliance, in the first reference use the full name as "Connectivity Standards Alliance (Alliance)". Use "Alliance" in subsequent references.
+
+### Public information
+
+This is a public website. Do not include any Alliance confidential information in these documents.
+
+### Acronyms
+- Write out acronyms on first use and include the acronym in parentheses
+  - First use of a Three Letter Acronym (TLA) should look similar to this.
+- Add new acronyms to the `.wordlist.txt` file for the spell checker
+- Add new acronyms glossary if appropriate
+
+### External links
+
+The handbook is meant to be used by a variety of users, not only Alliance members. This may include adopter and associate members as well as companies considering membership.
```

Comment:
```suggestion
The Handbook is meant to be used by a variety of users, not only Alliance members. This may include Adopter and Associate members as well as companies considering membership.
```